src/renderer/components/battle/OfflineBattleComponent.vue

```diff
@@ -127,7 +132,7 @@ import gridIcon from "@iconify-icons/mdi/grid";
 const mapListOpen = ref(false);
 const mapOptionsOpen = ref(false);
 const mapListOptions = useDexieLiveQuery(() => db.maps.toArray());
-const gameListOptions = useDexieLiveQuery(() => db.gameVersions.toArray());
+const gameListOptions = Array.from(gameStore.availableGameVersions.values());
```

Comment:
This should be computed property, afaiu this way it's just static value. Same issue in `DebugSidebar.vue`.